cli/flow-typed/npm/next_v9.x.x.js

```diff
@@ -0,0 +1,250 @@
+// flow-typed signature: 514646ec2cd38bf48c399f8b3ac0ea8c
+// flow-typed version: 071a03ea49/next_v9.x.x/flow_>=v0.104.x
+
+declare module "next" {
```

Comment:
Are we using Next.js on the CLI? :)

Reply:
It must be a dependency of a dependency, as per @Brianzchen recent work.

Reply:
I had a look, flowgen which is ships with flow has a dependency on `next`